src/utilities/TID300/Ellipse.js

```diff
@@ -30,23 +37,100 @@ export default class Ellipse extends TID300Measurement {
                     MeasurementUnitsCodeSequence: unit2CodingValue(areaUnit),
                     NumericValue: area
                 },
+                ContentSequence: buildContentSequence({
+                    graphicType: "ELLIPSE",
+                    graphicData: GraphicData,
+                    use3DSpatialCoordinates,
+                    referencedSOPSequence: ReferencedSOPSequence,
+                    referencedFrameOfReferenceUID: ReferencedFrameOfReferenceUID
+                })
+            }
+        ];
+
+        if (max) {
+            measurements.push({
```

Comment:
Again, all of these would become simpler if you just reference a structure class like MaximumValueReference(max, modalityUnit, annotationIndex)